devolving_music/views/get_song_pair.py

```diff
@@ -14,9 +16,20 @@ def get(self, _request, event: Event):
         voteable_submissions = ScoreSuite.get_voteable_submissions(event)
 
         if len(voteable_submissions) < 2:
-            return failure("Not enough songs have been submitted for this event.")
+            return failure(
+                "Not enough songs have been submitted for this event.")
 
-        sub1, sub2 = voteable_submissions[:2]
+        scores = SongScores(event)
+        scores_list = scores.get_scores()
+        info_list = SongScores.get_info_sort(list(scores_list.values()))
+        # make into a weighted info score
+        sub1_id = info_list[0].song_submission.id
+        sub2_id = scores.get_compare_submission_random(sub1_id)
+
+        # grab submission with least ammount of information
+        sub1 = SongSubmission.objects.get(id=sub1_id)
```

Comment:
```suggestion
        sub1 = info_list[0].song_submission
        sub2_id = scores.get_compare_submission_random(sub1.id)
```

We can save ourselves a database call.

Reply:
fair enough

Reply:
Fair enough but this isn't a good way to do it I'll adjust get_compare to return a song_submission